system-configuration-sys/src/lib.rs

```diff
@@ -16,6 +16,12 @@
 #![allow(non_upper_case_globals)]
 #![allow(non_snake_case)]
 
-extern crate core_foundation_sys;
+pub extern crate core_foundation_sys;
+pub extern crate libc;
+
+/// This is a temporary solution.
+pub type dispatch_queue_t = *mut ::std::os::raw::c_void;
 
 pub mod dynamic_store;
+pub mod network_configuration;
+pub mod preferences;
```

Comment:
Since there are sooo many functions and types I think that would make the crate root too messy. I'm also aiming to re-export the modules in the higher level modules, like how `core-foundation` does. See https://github.com/mullvad/system-configuration-rs/blob/master/system-configuration/src/dynamic_store.rs#L23

Reply:
Okay.